### DISCUSSION TITLE
SDR++ Version Question

### DISCUSSION THREAD

**Discussion by M0XAR** [AlexandreRouma/SDRPlusPlus#1383] [Q&A]
Which version do I need to install for my iteration of Linux Mint, which is 21.3 (Virginia)?

**Comment by feldmakg** [comment]
I have same question

**Comment by Mr-Precise** [comment]
You probably need a package for Ubuntu 22.04 LTS

**Comment by TeddJohnson** [comment]
it looks like Mint 21.3 does use Ubuntu 22.04 package base. 22.04 is "Jammy"


Per the readme for dependencies:
```sudo apt install libfftw3-dev libglfw3-dev libvolk2-dev libzstd-dev libairspyhf-dev libiio-dev libad9361-dev librtaudio-dev libhackrf-dev```

Then grab the "Jammy" package from here: https://www.sdrpp.org/nightly and install with 
```sudo dpkg -i sdrpp_ubuntu_jammy_amd64.deb```
